components/automatic_points_table/commons/automatic_points_table.lua

```diff
@@ -0,0 +1,106 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AutomaticPointsTable
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+---- This module creates a Points Table that automatically gets data from the prizepools
+---- of set tournaments
+
+local Class = require('Module:Class')
+-- local Table = require('Module:Table')
+
+local DivTable = require('Module:DivTable')
+local Json = require('Module:Json')
+local split = require('Module:StringUtils').split
+local getArgs = require('Module:Arguments').getArgs
+
+function DivTable.Row:create()
+  for _, cell in pairs(self.cells) do
+      cell:css('border-right', '1px solid #bbbbbb')

```

Comment:
General comment:
Module seems to be using 2 spaces as indent. The agreed code style is 1 tab (https://liquipedia.net/commons/Help:Scribunto#Code_style)